src/database/seeders/20220331192923-random-user.js

```diff
@@ -8,7 +8,7 @@ export default {
     const userPassword = await bcrypt.hash('altp6@random', salt);
     await queryInterface.bulkInsert('users', [
       {
-        id: uuidv4(),
+        id: "b66cfc7c-be2c-41f5-b459-e888bfe881a6",
```

Comment:
@SergeNm 
1. Why are you changing this a fixed value? 
2. Why is the assigned value equal to the id of an already existing user in our db? check `20220304143123-demo-users.js` for reference.

Reply:
this file has been created from [make trip request PR](https://github.com/atlp-rwanda/cod-be/blob/ft-make-trip-request-181414609/src/database/seeders/20220323103433-add-trip-requests.js)

Reply:
Every element of a model has a unique ID but it doesn't mean it is unique to the IDs of other models